src/pypromice/ressources/variable_aliases_GC-Net.csv

```diff
@@ -0,0 +1,78 @@
+﻿GEUS_name,old_name

```

Comment:
I would expect to have this data set specification located on the dataset repository or in out aws-l0 repository.

Reply:
Just like the other ressource files, it could be stored elsewhere but it requires some default values to be set in pypromice to be able to run without it. I suggest leaving relocation of ressource files to another PR.